index.html

```diff
@@ -41,60 +35,161 @@ <h2 class="primary_color">I'm a software developer</h2>
-              alt="Angelist contact"
-            />
+            <img class="social-icon" src="images/angellist.svg" alt="Angelist contact" />
           </a>
         </li>
       </ul>
     </section>
+
+    <div class="d-flex justify-content-center">
+      <img src="images/arrow.png" alt="down arrow" />
+    </div>
+
+    <!-- Works Section -->
+
+    <section class="grid_5x1">
+      <div>
+        <h2 class="secondary_color">My Recent <br />Works</h2>
+      </div>
+
+      <article class="grid_5x1">
+        <a href="#">
+          <img src="images/Snapshoot Portfolio.png" alt="Tonic Website Snapshoot" />
+        </a>
+        <h3 class="tertiary_color">Tonic</h3>
+        <ul class="d-flex">
+          <li class="li_works tertiary_color">CANOPY</li>
+          <li class="li_works tertiary_color">css</li>
+          <li class="li_works tertiary_color">Javascript</li>
+        </ul>
+        <a class="btn1" href="#">See Projects</a>
+      </article>
+
+      <article class="grid_5x1">
+        <a href="#">
+          <img
+            src="images/Snapshoot Portfolio-1.png"
+            alt="Multi-Post Stories Gain+Glory Snapshoot"
+          />
+        </a>
+        <h3 class="tertiary_color">Multi-Post Stories Gain+Glory</h3>
+        <ul class="d-flex justify-content-center">
+          <li class="li_works tertiary_color">Ruby on Rails</li>
+          <li class="li_works tertiary_color">css</li>
+          <li class="li_works tertiary_color">Javascript</li>
+          <li class="li_works tertiary_color">html</li>
+        </ul>
+        <a class="btn1" href="#">See Projects</a>
+      </article>
+
+      <article class="grid_5x1">
+        <a href="#">
+          <img src="images/Snapshoot Portfolio-2.png" alt="Tonic Website Snapshoot" />
+        </a>
+        <h3 class="tertiary_color">Tonic</h3>
+        <ul class="d-flex justify-content-between">
+          <li class="li_works tertiary_color">CANOPY</li>
+          <li class="li_works tertiary_color">css</li>
+          <li class="li_works tertiary_color">Javascript</li>
+        </ul>
+        <a class="btn1" href="#">See Projects</a>
+      </article>
+
+      <article class="grid_5x1">
+        <a href="#">
+          <img
+            src="images/Snapshoot Portfolio-3.png"
+            alt="Multi-Post Stories Gain+Glory Snapshoot"
+          />
+        </a>
+        <h3 class="tertiary_color">Multi-Post Stories</h3>
+        <ul class="d-flex justify-content-center">
+          <li class="li_works tertiary_color">Ruby on Rails</li>
+          <li class="li_works tertiary_color">css</li>
+          <li class="li_works tertiary_color">Javascript</li>
+          <li class="li_works tertiary_color">html</li>
+        </ul>
+        <a class="btn1" href="#">See Projects</a>
+      </article>
+    </section>
+
+    <!-- About me Section -->
+
+    <section class="d-flex flex-column align-items-center">
+      <article class="d-flex flex-column align-items-center border about_me">
```

Comment:
- [ ] Please remove the border around this article so that it matches the figma tmeplate.
### Your design
![3](https://user-images.githubusercontent.com/94551484/205329301-fd37ed91-d203-442e-9e98-b34a0d1ea92a.png)
### Figma template
![f3](https://user-images.githubusercontent.com/94551484/205329354-7846bf6b-afb3-4548-9da5-931454047c1b.png)

Reply:
:red_circle:  As requested by the previous CR Kindly remove the border around this block to match the Figma design.

Your Design  
<img src="https://user-images.githubusercontent.com/94551484/205329301-fd37ed91-d203-442e-9e98-b34a0d1ea92a.png">

Figma Design
<img src="https://user-images.githubusercontent.com/94551484/205329354-7846bf6b-afb3-4548-9da5-931454047c1b.png">